From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jeannt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jeannt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

rayne-wiselman committed:
adding new migration planner docs

rayne-wiselman committed:
Update index.yml

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update how-to-set-up-rest-api.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update how-to-set-up-rest-api.md

rjagiewich committed:
Merge pull request #25587 from rayne-wiselman/raynemigrate-october17

adding new migration planner docs

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

rayne-wiselman committed:
updated versions of migrate articles

rayne-wiselman committed:
Update how-to-modify-assessment.md

rayne-wiselman committed:
Update toc.yml

v-anpasi committed:
Bringing even with master.

rayne-wiselman committed:
Update migrate-overview.md

rayne-wiselman committed:
Update concepts-assessment-calculation.md

rayne-wiselman committed:
Update how-to-create-a-group.md

v-anpasi committed:
Bringing even with master.

rayne-wiselman committed:
Update how-to-create-group-dependencies.md

rayne-wiselman committed:
Update how-to-create-group-machine-dependencies.md

rayne-wiselman committed:
Update how-to-modify-assessment.md

rayne-wiselman committed:
Update how-to-tag-v-center.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
removing extraneous file, adding missing file

rayne-wiselman committed:
Update tutorial-assessment-vmware.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
adding new file

rayne-wiselman committed:
Update troubleshooting-general.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update toc.yml

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

rayne-wiselman committed:
Merge branch 'raynemigrate-october24' of https://github.com/rayne-wiselman/azure-docs-pr into raynemigrate-october24

rayne-wiselman committed:
Update concepts-dependency-visualization.md

rayne-wiselman committed:
Update concepts-assessment-calculation.md

rayne-wiselman committed:
Update how-to-modify-assessment.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update troubleshooting-general.md

rayne-wiselman committed:
Update tutorial-assessment-vmware.md

v-anpasi committed:
Bringing even with master.

rayne-wiselman committed:
updating images

rayne-wiselman committed:
Update concepts-assessment-calculation.md

rayne-wiselman committed:
updating two graphic files

rayne-wiselman committed:
Update how-to-create-a-group.md

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

Commit:
edit pass: active-directory-aadconnect-pass-through-authentication-quick-start

Commit:
edit pass: active-directory-aadconnect-pass-through-authentication-quick-start

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

carolinacmoravia committed:
C6851: Hardcoded EN locale in link 

Hello, @bburns,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6870: Hardcoded EN locale in link

Hello, @botatoes, 
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6877: Hardcoded EN locale in link 

Hello, @botatoes,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

v-anpasi committed:
Bringing even with master.

carolinacmoravia committed:
C6854: Hardcoded EN locale in link

Hello, @BYHAM,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6880: Hardcoded EN locale in link

Hello, @BYHAM,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6873: Hardcoded EN locale in link 

Hello, @cawa,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6871: Hardcoded EN locale in link 

Hello, @curtand,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6882: Hardcoded EN locale in link

Hello, @curtand,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6858: Hardcoded EN locale in link 

Hello, @MarkMcGeeAtAquent,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6856: Hardcoded EN locale in link

Hello, @rwike77,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

carolinacmoravia committed:
C6881: Hardcoded EN locale in link

Hello, @SnehaGunda, 
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6864: Hardcoded EN locale in link

Hello, @tamram,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6860: Hardcoded EN locale in link

Hello, @tompratt-AQ, 
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

v-anpasi committed:
Bringing even with master.

rayne-wiselman committed:
Update troubleshooting-general.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update how-to-modify-assessment.md

rayne-wiselman committed:
Update migrate-overview.md

rayne-wiselman committed:
Update migrate-overview.md

rayne-wiselman committed:
Update troubleshooting-general.md

rayne-wiselman committed:
Update migrate-overview.md

rayne-wiselman committed:
updating files for public preview

rayne-wiselman committed:
Update how-to-get-migration-tool.md

rayne-wiselman committed:
Update how-to-get-migration-tool.md

rayne-wiselman committed:
Update how-to-scale-assessment.md

rayne-wiselman committed:
Update how-to-get-migration-tool.md

tfosmark committed:
Merge pull request #26182 from rayne-wiselman/raynemigrate-october24

updated versions of migrate articles

rayne-wiselman committed:
updating files for November 23 merge to master

rayne-wiselman committed:
Update index.yml

ktoliver committed:
Merge pull request #27735 from rayne-wiselman/raynemigrate-november14

updating files for merge to master next week

v-anpasi committed:
Bringing even with master.

genlin committed:
add a new article

v-anpasi committed:
Bringing even with master.

genlin committed:
correc the error

genlin committed:
add new section

genlin committed:
correct typo

genlin committed:
add file into toc

genlin committed:
update the date

genlin committed:
add blod

rayne-wiselman committed:
adding last changes

rayne-wiselman committed:
adding overview article

rayne-wiselman committed:
Update how-to-modify-assessment.md

rayne-wiselman committed:
Update tutorial-assessment-vmware.md

rayne-wiselman committed:
Merge branch 'raynemigrate-november16' of https://github.com/rayne-wiselman/azure-docs-pr into raynemigrate-november16

rayne-wiselman committed:
Update migrate-overview.md

rayne-wiselman committed:
Update tutorial-assessment-vmware.md

rayne-wiselman committed:
updating troubleshooting doc

rayne-wiselman committed:
Update how-to-scale-assessment.md

ktoliver committed:
Merge pull request #27994 from rayne-wiselman/raynemigrate-november16

Raynemigrate november16

carolinacmoravia committed:
P: Broken table at line 29

Hello, @rwike77, 

Localization team has reported source content issues that causes localized version to have broken format compared to en-us version. 
Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

v-anpasi committed:
Bringing even with master.

Commit:
Updating recommendation

v-alje committed:
Bringing even with master.

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ci68377

Commit:
Merge pull request #2 from MicrosoftDocs/master

Merge from remote

Commit:
update images

rayne-wiselman committed:
couple of changes

genlin committed:
correct the article

genlin committed:
correct the typo

rayne-wiselman committed:
Update migrate-overview.md

rayne-wiselman committed:
Update migrate-overview.md

PRMerger17 committed:
Merge pull request #28238 from rayne-wiselman/raynemigrate-november20

Raynemigrate november20

ruoxinCheng committed:
Update the USQL document

Commit:
edit pass: active-directory-aadconnect-pass-through-authentication-quick-start

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into four-active-directory-aadconnect

Commit:
edit pass: active-directory-aadconnect-pass-through-authentication-how-it-works

ruoxinCheng committed:
fix format

ruoxinCheng committed:
fix spelling

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ci68377

v-alje committed:
Bringing even with master.

ruoxinCheng committed:
Change a png

ruoxinCheng committed:
Add storage passage

SnehaGunda committed:
Updates to register topic

ShawnJackson committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

ShawnJackson committed:
Update active-directory-aadconnect-pass-through-authentication-faq.md

ShawnJackson committed:
Update active-directory-aadconnect-pass-through-authentication-how-it-works.md

ShawnJackson committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

Commit:
fix error on supported API

Commit:
Update stream-analytics-documentdb-output.md

Commit:
Update stream-analytics-documentdb-output.md

sanjeev3 committed:
Update video-moderation-api.md

Commit:
Minor version updates and edits

SnehaGunda committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into policy

sanjeev3 committed:
Update video-moderation-api.md

Commit:
update content, remove images

Commit:
fix image links

ruoxinCheng committed:
Add new doc for python,R and CS

AnoopVasudavan committed:
modifying dates

ruoxinCheng committed:
Update Jenny feedback

ruoxinCheng committed:
fix format

rayne-wiselman committed:
last minute update on files before public preview

rayne-wiselman committed:
fixing toc and index

rayne-wiselman committed:
updating 2 graphics

rayne-wiselman committed:
updating graphics again

rayne-wiselman committed:
updating graphics size

GitHubber17 committed:
Merge pull request #28435 from rayne-wiselman/raynemigrate-november22

last minute update on files before public preview

v-alje committed:
Merge pull request #28455 from MicrosoftDocs/master

Merge release-migration-planner to master for public preview

carolinacmoravia committed:
M15129: Extra word in a sentence: "and"

Hello, @iainfoulds, 
Translator has reported possible source content issue.  It seems that "and" is unncessary for the reported sentence. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

carolinacmoravia committed:
M15207: Typo mistake: "area"

Hello, @mimig1,

Translator has reported possible source content issue. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

SnehaGunda committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TOC

SnehaGunda committed:
updating TOC

Commit:
Updated diagnostics article

Commit:
Fixed headings

Commit:
Fixed code blocks

Commit:
Final tweaks

Commit:
Final tweaks

Commit:
Final tweaks

Commit:
Final tweaks

Commit:
added powershell alias to code blocks

Commit:
added json alias and fixed typo

Commit:
adjusted numbering and indent

Commit:
task 625: SEO and grammar edits

Lisaco88 committed:
Merge pull request #28485 from jeffgilb/jeff1122

Updated diagnostics article

v-alje committed:
Merge pull request #28458 from MicrosoftDocs/release-migration-planner

Release migration planner

Commit:
devices06

rmca14 committed:
Merge pull request #28489 from SnehaGunda/TOC

Toc

PRMerger16 committed:
Merge pull request #28501 from dksimpson/dean-625

task 625: SEO and grammar edits

Commit:
Merge pull request #28502 from MarkusVi/devices06

devices06

Commit:
update date

ruoxinCheng committed:
fix blocking issue

PRMerger12 committed:
Merge pull request #28430 from AnoopVasudavan/master

modifying dates

apwestgarth committed:
Update azure-stack-app-service-deploy-offline.md

Corrected default name of resource group

apwestgarth committed:
Update azure-stack-tutorial-app-service.md

Removed references to mobile apps as they are not included in the GA release of Azure App Service on Azure Stack

apwestgarth committed:
Update azure-stack-tutorial-app-service.md

Removed additional the to improve grammar

PRMerger13 committed:
Merge pull request #28508 from apwestgarth/patch-1

Update azure-stack-app-service-deploy-offline.md

Commit:
Merge pull request #28509 from apwestgarth/patch-2

Update azure-stack-tutorial-app-service.md

Commit:
Updated content

- Added Pronunciation Characters
- Added Encoding info

Commit:
added local content

PRMerger19 committed:
Merge pull request #28510 from PanosPeriorellis/master

Edit Document on pronunciation of Custom vocabulary

genlin committed:
add new linux issue

bsiva committed:
Updating dates

PRMerger19 committed:
Merge pull request #28512 from genlin/ci001

add new linux issue

bsiva committed:
Updates based on accrolinx suggestions.

bsiva committed:
update

PRMerger20 committed:
Merge pull request #28513 from bsiva/master

Updating dates

gsacavdm committed:
Add language hints

PRMerger15 committed:
Merge pull request #28515 from gsacavdm/patch-1

Add language hints

linda33wj committed:
Fix SF copy sample JSON

linda33wj committed:
Refine Oracle data type support

ruoxinCheng committed:
change a video scrennshot

linda33wj committed:
Fix the unexpected link

Commit:
Merge pull request #28517 from linda33wj/master

Fix SF copy sample JSON

Commit:
Merge pull request #27111 from carolinacmoravia/patch-428

C6870: Hardcoded EN locale in link

PRMerger15 committed:
Merge pull request #27106 from carolinacmoravia/patch-427

C6851: Hardcoded EN locale in link

Commit:
Merge pull request #27178 from carolinacmoravia/patch-430

C6854: Hardcoded EN locale in link

PRMerger17 committed:
Merge pull request #27187 from carolinacmoravia/patch-434

C6873: Hardcoded EN locale in link

Commit:
Merge pull request #27188 from carolinacmoravia/patch-435

C6871: Hardcoded EN locale in link

PRMerger16 committed:
Merge pull request #27189 from carolinacmoravia/patch-436

C6882: Hardcoded EN locale in link

PRMerger16 committed:
Merge pull request #27461 from carolinacmoravia/patch-453

C6864: Hardcoded EN locale in link

PRMerger19 committed:
Merge pull request #27462 from carolinacmoravia/patch-454

C6864: Hardcoded EN locale in link

Commit:
Merge pull request #27463 from carolinacmoravia/patch-455

C6860: Hardcoded EN locale in link

Commit:
Merge pull request #28029 from carolinacmoravia/patch-464

P: Broken table at line 29

Commit:
Merge pull request #27241 from carolinacmoravia/patch-451

C6856: Hardcoded EN locale in link

PRMerger14 committed:
Merge pull request #27180 from carolinacmoravia/patch-431

C6880: Hardcoded EN locale in link

PRMerger10 committed:
Merge pull request #27114 from carolinacmoravia/patch-429

C6877: Hardcoded EN locale in link

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ci68377

genlin committed:
correct format issue

genlin committed:
correct format issue

GitHubber17 committed:
Merge pull request #28419 from shizn/master

Update devkit happy path

ruoxinCheng committed:
Update python and R doc

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
added comment to custom pronunciation file

added explanation

Commit:
Fixed broken link

added link to client libraries

v-alje committed:
Merge pull request #28533 from MicrosoftDocs/FromPublicRepo

From public repo

bwren committed:
AI Analytics

Commit:
minor edits

edits in the custom pronunciation file

PRMerger17 committed:
Merge pull request #28534 from PanosPeriorellis/master

Updated 2 fiels

PRMerger13 committed:
Merge pull request #28480 from carolinacmoravia/patch-475

M15207: Typo mistake: "area"

v-geberr committed:
touch

Lisaco88 committed:
Merge pull request #27983 from genlin/ci68377

storage-migration-faq.md

v-alje committed:
Merge pull request #28540 from v-geberr/1127-2

touch

Lisaco88 committed:
Merge pull request #28335 from ruoxinCheng/usqlvscode

Update doc for Usql vscode

Lisaco88 committed:
Merge pull request #27197 from carolinacmoravia/patch-443

C6858: Hardcoded EN locale in link

Lisaco88 committed:
Merge pull request #28468 from carolinacmoravia/patch-472

M15129: Extra word in a sentence: "and"

PRMerger19 committed:
Merge pull request #28535 from bwren/ai

AI Analytics link fix

Commit:
Update stream-analytics-documentdb-output.md

MarkMcGeeAtAquent committed:
added link

PRMerger20 committed:
Merge pull request #28403 from szarkos/szark-20171121

Minor version updates and edits

PRMerger17 committed:
Merge pull request #28546 from sanjeev3/master

Updated video API article

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-lakes

Commit:
Merge pull request #28391 from JSeb225/patch-18

fix error on supported API

PRMerger12 committed:
Merge pull request #28158 from JSeb225/patch-16

Updating recommendation

ShawnJackson committed:
Merge pull request #26859 from LoriMerrick/four-active-directory-aadconnect

edit pass: four active-directory-aadconnect-pass-through-authentication

PRMerger13 committed:
Merge pull request #28550 from MarkMcGeeAtAquent/data-lakes

Data Lake Analytics - add link
